Source/WebCore/dom/Document.h

```diff
@@ -1682,6 +1682,9 @@ class Document
 
     void performPendingViewTransitions();
 
+    bool renderingSuppressedForViewTransition() const;
```

Comment:
This should be `isRenderingSuppressedForViewTransition()` or `renderingIsSuppressedForViewTransition()`